src/layouts/MainLayout.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function MainLayout() {
```

Comment:
could we use an arrow function instead ? can we delete the unnecessary return statement ?